test/spec/modules/prebidServerBidAdapter_spec.js

```diff
@@ -1477,6 +1477,7 @@ describe('S2S Adapter', function () {
       adapter.callBids(REQUEST, BID_REQUESTS, addBidResponse, done, ajax);
       const requestBid = JSON.parse(server.requests[0].requestBody);
       expect(requestBid.site).to.exist.and.to.be.a('object');
+      expect(requestBid.app).to.not.exist;
```

Comment:
I don't think this is enough to test #7329. The test should say "if both app and site are set, only app should be set in the request". The setup of this test only sets up `site` so it's not covering the conflict.

I was not aware that `app` and `site` are mutually exclusive in ORTB; I think this requires changes in the `ortbConverter` logic.

Reply:
updating 👍

Reply:
I think so as well @dgirardi ; also 'dooh' is exclusive to these